README.md

```diff
@@ -7,28 +7,39 @@ Esta é uma implementação em javascript que atende aos requisitos do
 
 ### Client
 
-A forma mais simples é através do generator `generateFetchEventByName`
-
-```js
-import client from "@guiabolsobr/events-protocol/lib/client";
+A basta apenas intancia um EventClient e enviar um evento.
+
+```ts
+import EventsClient from "src/client/client";
```

Comment:
No caso de quem for usar a lib, o import n vai ser esse, e sim `@guiabolsobr/events-protocol/lib/client`, não?